### PR TITLE
samples/bookinfo: easier access to logs

### LINKED ISSUE #10985: Nice to have `kubectl logs productpage` output
**Describe the feature request**
Would be great to be able to use `kubectl logs productpage-v1 -c productpage`

Looks like the sample dumps it's output to a file called microservice.log in the root dir.  Would be great to have that tailed out to stdout to allow logs to pick it up.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by paranoidaditya**
closes #10985

### CODE REVIEW

File: samples/bookinfo/src/productpage/Dockerfile [hunk 1]
```diff
@@ -29,4 +29,4 @@ COPY requirements.txt /opt/microservices/
 EXPOSE 9080
 WORKDIR /opt/microservices
 RUN python -m unittest discover
-CMD python productpage.py 9080
+CMD touch /opt/microservices/microservice.log && tail -f /opt/microservices/microservice.log & python productpage.py 9080
```
Comment (vadimeisenbergibm) [inline samples/bookinfo/src/productpage/Dockerfile]: missing `&&` after `tail -f /opt/microservices/microservice.log`, note the comment https://github.com/istio/istio/issues/10985#issuecomment-459783710

Comment (paranoidaditya) [reply samples/bookinfo/src/productpage/Dockerfile]: Are we not just backgrounding the `tail ...`? && would wait for tail to complete, which would never?

Comment (vadimeisenbergibm) [reply samples/bookinfo/src/productpage/Dockerfile]: I see, you are right.

**Comment by paranoidaditya** [comment]
@esnible waiting on that `/approve` 🙂

**Comment by esnible** [comment]
Any idea how to get the test to pass?  We could try rebasing against the latest `master` or against `release-1.1`.

**Comment by paranoidaditya** [comment]
@esnible, rebased against current master and pushed but ....

**Comment by esnible** [comment]
> @esnible, rebased against current master and pushed but ....

The required tests all pass so I think you are good.  All we need is someone in the _samples/OWNERS_ file to `/lgtm` again.

**Pull request merged**